HW_5/task_3.py

```diff
@@ -0,0 +1,129 @@
+"""
+Напишите игру "Крестики-нолики".
+"""
+from random import randint
+import os
+
+
+def print_board(array: list):
+    print('|', end=' ')
+    print(*range(len(array)), sep=' | ', end=' | \n')
+    print('-' * 13)
+    for i, line in enumerate(array):
+        print('|', end='')
+        print(*line, sep='|', end=f'| {i} \n')
+        print('-' * 13)
+
+
+def check(array):
+    if array[0][0] == array[0][1] == array[0][2] != " " or \
+            array[1][0] == array[1][1] == array[1][2] != " " or \
+            array[1][0] == array[1][1] == array[1][2] != " " or \
+            array[0][0] == array[1][0] == array[2][0] != " " or \
+            array[1][0] == array[1][1] == array[1][2] != " " or \
+            array[2][0] == array[2][1] == array[2][2] != " " or \
+            array[0][0] == array[1][1] == array[2][2] != " " or \
+            array[0][2] == array[1][1] == array[2][0] != " ":
+        return True
+    return False
+
+
+if __name__ == '__main__':
+    lst = [[' ', ' ', ' '], [' ', ' ', ' '], [' ', ' ', ' ']]
+    print('Начинаем игру крестики-нолики')
+    players = ['0', 'X']
+    turn = randint(0, 1)
+    player = players[turn]  # turn = ot turn
+    print_board(lst)
+    while True:
+        print(f'ходят {player}')
+        row, col = [int(i) for i in input(
+            'Укажите строку и столбец через пробел: ').split()]
+        os.system('cls')
+        lst[row][col] = player
+        print(lst)
```

Comment:
Вот это не нужно иначе будет выводиться так
![image](https://user-images.githubusercontent.com/49741194/207253905-b014bb57-2006-4f9c-97f6-ad4d9c7618ad.png)
```suggestion
        # print(lst)
```